Tests/ApolloTests/StarWars/API.swift

```diff
@@ -131,36 +159,36 @@ public final class HeroAndFriendsNamesQuery: GraphQLQuery {
   public struct Data: GraphQLMappable {
     public let hero: Hero?
 
-    public init(reader: GraphQLResultReader) throws {
-      hero = try reader.optionalValue(for: Field(responseName: "hero", arguments: ["episode": reader.variables["episode"]]))
+    public init(values: [Any?]) {
+      hero = values[0] as! Hero?
     }
 
     public struct Hero: GraphQLMappable {
       public let __typename: String
       public let name: String
       public let friends: [Friend?]?
 
-      public init(reader: GraphQLResultReader) throws {
-        __typename = try reader.value(for: Field(responseName: "__typename"))
-        name = try reader.value(for: Field(responseName: "name"))
-        friends = try reader.optionalList(for: Field(responseName: "friends"))
+      public init(values: [Any?]) {
+        __typename = values[0] as! String
+        name = values[1] as! String
+        friends = values[2] as! [Friend?]?
       }
 
       public struct Friend: GraphQLMappable {
         public let __typename: String
         public let name: String
 
-        public init(reader: GraphQLResultReader) throws {
-          __typename = try reader.value(for: Field(responseName: "__typename"))
-          name = try reader.value(for: Field(responseName: "name"))
+        public init(values: [Any?]) {
+          __typename = values[0] as! String
+          name = values[1] as! String
         }
       }
     }
   }
 }
 
 public final class HeroAndFriendsNamesWithIDsQuery: GraphQLQuery {
-  public static let operationDefinition =
+  public static let operationString =
     "query HeroAndFriendsNamesWithIDs($episode: Episode) {" +
```

Comment:
I really like that the generated output keeps the line breaks for the queries. It’s not obvious, but it’s a small DX thing that makes me happy 😊